api/endpoints/move.md

```diff
@@ -0,0 +1,297 @@
+# Group Move
+Move-related resources of *iOffice API*
+
+## Move [/move/requests///{id}]
```

Comment:
I thought to address this, and the next comment, I would match the style of the user look up, where the user collection call still lives in the collection sub header, but now contains different endpoints, rather than listing those out individually.